surveymonkey/api_surveymonkey.py

```diff
@@ -0,0 +1,108 @@
+"""
+This Api class allows to authenticate and make request to the surveymonkey API_BASE
+"""
+import logging
+import requests
+
+from django.core.cache import cache
+from requests_oauthlib import OAuth2Session
+from oauthlib.oauth2 import BackendApplicationClient
+
+LOG = logging.getLogger(__name__)
+API_BASE = "https://api.surveymonkey.com"
+CACHE_TIMEOUT = 86400
+
+
+class ApiSurveyMonkey(object):
+    """
+    Class with the necessary methods to make request to surveymonkey API_BASE
+    """
+    def __init__(self, client_id, client_secret):
+        self.session = requests.Session()
+
+        key = "{}-{}".format("api_survey_monkey", client_id)
+        headers = cache.get(key)
+
+        if headers is None:
```

Comment:
I think `if not headers` would be better

Reply:
ok